CONTRIBUTING.md

```diff
@@ -14,6 +14,8 @@ If you have a local WordPress environment, you can clone this repository into yo
 
 Some good options for a local WordPress development environment include <a href="https://varyingvagrantvagrants.org/">VVV</a> and <a href="https://www.mamp.info/">Mamp</a>.
 
+> Once the plugin is intalled, and you have built it through `npm` go to the post list and hover over a post row to click `Edit in Gutenberg` link. That should do it for initial testing.
```

Comment:
@aduth I think the admin bar menu gets ignored, pretty easily. But feel free to edit the text before you merge this one. Or let me know if I should do it.